api/app/signals/apps/signals/migrations/0161_signal_user_HISTORY.py

```diff
@@ -0,0 +1,173 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2019 - 2022 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
+from django.db import migrations
+
+history_view = """
```

Comment:
Please double check that the new history implementation (which is currently active on Amsterdam PROD and ACC) if this will not be an issue.

https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/history/services/signal_log.py#L152